orttraining/orttraining/python/training/_ortmodule_graph_execution_manager.py

```diff
@@ -214,14 +218,17 @@ def _get_exported_model(self, *inputs, **kwargs):
             _io.parse_outputs_for_onnx_export_and_extract_schema(self._original_module, inputs, kwargs)
         self._input_info.dynamic_axes.update(output_dynamic_axes)
 
+        # FlattenedModule needs _InputInfo to expand user input from *args to *args + **kwargs
+        self._flattened_module._input_info = self._input_info
```

Comment:
Minor nit and optional: I wonder if we could refactor a little so that we don't have to mutate the flattened module. (It might make life easier in case we end up with multiple concurrent callers, or want to support caching graphs for multiple different input infos.)

Reply:
That is actually a bit tricky. I will track on a second PR. "input_info.schema" is also updated within ORTModule scope after instantiation.
Is that ok?

Reply:
Oh yes, fine to do in a later PR!